humanoid_league_team_communication/config/team_communication_config.yaml

```diff
@@ -1,28 +1,31 @@
-# The port to be used
-port: 12321
+# Only used if local_mode is false
+target_host: 127.0.0.1
+target_port: 3737
+receive_port: 3737
+
+# Only used when target_host is 127.0.0.1 (local mode)
+# the team communication will bind to one of these ports and send to the other ports, depending on its bot_id
+local_target_ports:
+  - 4001
+  - 4002
+  - 4003
+  - 4004
 
 # Rate of published messages in Hz
 rate: 1
 
 # average walking speed in [m/s]
-avg_walking_speed: 0.01
-
-# set the color of your team
-# 2: red
-# 3: blue
-# other values are invalid
-team_color: 2
-
-#lifetime of data [s]
-lifetime: 3
+avg_walking_speed: 0.2
```

Comment:
I think we should communicate our `time_to_ball` in the team comm. This is essential to easily determine which player is the fastest at the ball. This parameter should be used for this purpose.

Reply:
That is exactly what is done in line 379

Reply:
Oh I searched for it last night. Maybe I was too tired